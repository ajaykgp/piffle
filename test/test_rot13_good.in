The quick brow fox jumps over the lazy dog.
Gur dhvpx oebj sbk whzcf bire gur ynml qbt!
