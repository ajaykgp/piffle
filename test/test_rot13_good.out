Gur dhvpx oebj sbk whzcf bire gur ynml qbt!
The quick brow fox jumps over the lazy dog!
